### DISCUSSION TITLE
Doesn't actually rate limit?

### DISCUSSION THREAD

**Discussion by RichieMcMullen** [danharrin/livewire-rate-limiting#11] [Q&A] [answered]
This doesn't actually rate limit the request. It just prevents the action from being completed, posting a comment etc....

For example if I rate limit posting a comment it prevents the comment being posted, but it still has to make the request to the server to trigger the rate limit. 

Does this not leave this package irrelevant ?

**Comment by danharrin** [comment, marked as answer]
The whole point is that it prevents your application from being brute forced / spammed, not your server. For instance, it will stop an attacker trying to guess a password on your login form, or spamming your comments section.

This was never advertised as DoS protection or anything, I would advise using Cloudflare or another service for that.

**Reply by RichieMcMullen** [reply]
Ah ok thanks for clearing that up. 👍
